docs/CONTROLFLOW.md

```diff
@@ -0,0 +1,343 @@
+| 1              | 1024         | 0          | 95          |
+| 1              | 0            | 10         | 62          |
+| 1              | 10           | 10         | 92          |
+| 1              | 100          | 10         | 125         |
+| 1              | 1024         | 10         | 125         |
+| 1              | 0            | 100        | 332         |
+| 1              | 10           | 100        | 362         |
+| 1              | 100          | 100        | 395         |
+| 1              | 1024         | 100        | 395         |
+| 2              | 0            | 0          | 123         |
+| 2              | 10           | 0          | 233         |
+| 2              | 100          | 0          | 354         |
+| 2              | 1024         | 0          | 354         |
+| 2              | 0            | 10         | 233         |
+| 2              | 10           | 10         | 343         |
+| 2              | 100          | 10         | 464         |
+| 2              | 1024         | 10         | 464         |
+| 2              | 0            | 100        | 1223        |
+| 2              | 10           | 100        | 1333        |
+| 2              | 100          | 100        | 1454        |
+| 2              | 1024         | 100        | 1454        |
+| 3              | 0            | 0          | 486         |
+| 3              | 10           | 0          | 916         |
+| 3              | 100          | 0          | 1389        |
+| 3              | 1024         | 0          | 1389        |
+| 3              | 0            | 10         | 916         |
+| 3              | 10           | 10         | 1346        |
+| 3              | 100          | 10         | 1819        |
+| 3              | 1024         | 10         | 1819        |
+| 3              | 0            | 100        | 4786        |
+| 3              | 10           | 100        | 5216        |
+| 3              | 100          | 100        | 5689        |
+| 3              | 1024         | 100        | 5689        |
+| 4              | 0            | 0          | 1937        |
+| 4              | 10           | 0          | 3647        |
+| 4              | 100          | 0          | 5528        |
+| 4              | 1024         | 0          | 5528        |
+| 4              | 0            | 10         | 3647        |
+| 4              | 10           | 10         | 5357        |
+| 4              | 100          | 10         | 7238        |
+| 4              | 1024         | 10         | 7238        |
+| 4              | 0            | 100        | 19037       |
+| 4              | 10           | 100        | 20747       |
+| 4              | 100          | 100        | 22628       |
+| 4              | 1024         | 100        | 22628       |
+
+
+
+### Caveats
+
+* Obfuscation breaks the lazy iteration over maps. See: [ssa2ast/polyfill.go](../internal/ssa2ast/polyfill.go)
+* Generic functions not supported
```

Comment:
what happens if I try to use control flow obfuscation on them? silently ignored, or error? you should probably say.